src/admin-views/settings/tabs/general/general-editing.php

```diff
@@ -6,8 +6,8 @@
  * @since 6.7.0
  */
 
-$is_missing_aggregator_license_key = '' === get_option( 'pue_install_key_event_aggregator' );
-$should_hide_upsell                = tec_should_hide_upsell();
+$is_missing_aggregator_license_key = true; // '' === get_option( 'pue_install_key_event_aggregator' );
+$should_hide_upsell                = false; //tec_should_hide_upsell();
```

Comment:
I don't think you meant to save this.
```suggestion
$is_missing_aggregator_license_key = '' === get_option( 'pue_install_key_event_aggregator' );
$should_hide_upsell                = tec_should_hide_upsell();
```

Reply:
whups - thanks!